circle.yml

```diff
@@ -4,3 +4,13 @@ dependencies:
 test:
   override:
     - yarn test
+
+deployment:
+  release:
+    tag: /v[0-9]+(\.[0-9]+)*/
+    commands:
+        - /bin/bash publishTag
```

Comment:
Explicitly calling bash here is redundant, given the shebang in the bash script.